### PR TITLE
Address feedback from internal test

### PR DESCRIPTION
**Pull request by MoLi7**

### CODE REVIEW

File: CalculateEmissionFactors.Rmd [hunk 1]
```diff
@@ -92,3 +92,6 @@ table <- table[!substr(table[, paste(params$modeltype, "Code")], 1, 1)%in%c("S",
 write_to_filename <- paste0("SupplyChainGHGEmissionFactors_", modelname, ".csv")
 write.csv(table, write_to_filename, row.names = FALSE)
 ```
+
+Saved `r write_to_filename`.
```
Comment (WesIngwersen) [inline CalculateEmissionFactors.Rmd:96]: This at least gives the user the name of this file, although I see it the markdown converts the '_' to '\'

**Review by WesIngwersen — APPROVED** [review]
Works fine for me still - i added a couple more cosmetic changes

**Comment by MoLi7** [comment]
> Works fine for me still - i added a couple more cosmetic changes

Same here. I added a few more changes in README to further clarify usage options.

File: README.Rmd [hunk 2]
```diff
@@ -16,16 +16,18 @@ Code to produce the [Supply Chain GHG Emission Factors](https://cfpub.epa.gov/si
...
 
 ## Usage
 
 The code in this package can be used to generate the USEEIO-based Supply Chain GHG Emission Factors, based on options selected by the user at run-time. 
 
-These factors can be generated by [creating a project from the existing directory within Rstudio](https://support.rstudio.com/hc/en-us/articles/200526207-Using-RStudio-Projects) by selecting the unzipped local directory of files in this repository. Once created, open
-the _CalculateEmissionFactors.Rmd_ in the editor, click on *Knit* and select *Knit with Parameters…*.
+If users choose to download a zip file of the code, they can unzip the file then [create a project from the existing directory within RStudio](https://support.rstudio.com/hc/en-us/articles/200526207-Using-RStudio-Projects) by selecting the unzipped local directory of files.
+
+If users choose to clone this repository, they can [create a project by cloing the repository with RStudio](https://resources.github.com/whitepapers/github-and-rstudio/#:~:text=Clone%20the%20repository%20with%20RStudio).
```
Comment (WesIngwersen) [inline README.Rmd:28]: I think its should rather be

"If users have git installed, the users can create a project by cloning the repository with RStudio"

File: README.Rmd [hunk 3]
```diff
@@ -16,16 +16,18 @@ Code to produce the [Supply Chain GHG Emission Factors](https://cfpub.epa.gov/si
...
+
+Once a project is created, double click to open it, then open the *CalculateEmissionFactors.Rmd* in the editor, click on *Knit* and select *Knit with Parameters…*.
```
Comment (WesIngwersen) [inline README.Rmd:30]: I don't have to open the project by double-clicking ... once its created its automatically opened

File: README.Rmd (hunk 2 above)
Comment (MoLi7) [reply README.Rmd:28]:
```suggestion
If users have git installed, they can [create a project by cloning the repository with RStudio](https://resources.github.com/whitepapers/github-and-rstudio/#:~:text=Clone%20the%20repository%20with%20RStudio).
```

File: README.Rmd (hunk 3 above)
Comment (MoLi7) [reply README.Rmd:30]:
```suggestion
Once a project is created and opened, then open the *CalculateEmissionFactors.Rmd* in the editor, click on *Knit* and select *Knit with Parameters…*.
```

Comment (MoLi7) [reply README.Rmd:30]: Under the 'clone' option, users have to double click to open it.

**Pull request merged**